test/config/config.test.js

```diff
@@ -43,6 +43,19 @@ function getUseAgentLogs(stdout) {
     .map((l) => l.replace(prefix, ''));
 }
 
+/**
+ * Returns the warnings emitted by the logger
```

Comment:
nit: The description isn't quite accurate about it being specifically about "warn" level.